sklearn/cluster/dbscan_.py

```diff
@@ -208,6 +212,9 @@ class DBSCAN(ClusterMixin, BaseEstimator):
         important DBSCAN parameter to choose appropriately for your data set
         and distance function.
 
+        Note that there is no good default value for this parameter. An
+        optimal value depends on the data at hand as well as the used metric.
```

Comment:
We should also indicate that the default is 0.5 (which is generally bad) and that a warning will be raised unless the value is explicitly set.